v2/git-initializer-windows/Dockerfile

```diff
@@ -19,6 +19,12 @@ RUN go build \
 
 FROM mcr.microsoft.com/windows/nanoserver:1809
 
+COPY --chown=ContainerUser:ContainerUser v2/git-initializer/ssh_config /Users/ContainerUser/.ssh/config
+COPY --from=builder /git /git/
```

Comment:
We got super lucky that the `golang:1.18.1-windowsservercore-1809` uses the "portable" git installed that can just be copied and over and used. It saved us a lot of hassle.